Makefile

```diff
@@ -21,16 +21,16 @@ ctags:
 		> tags
 
 build-example: $(OUTPUT)
-	pulp build -I example --to $(OUTPUT)/example.js
+	npx spago bundle-app --path $(example) --to $(OUTPUT)/example.js --minify
```

Comment:
Isn't bundling functionality going away?

Reply:
Native bundling is going away, and instead Spago will call out to esbuild under the hood (if it's installed), or you can swap in another bundler.